java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/IAtomicReferenceTest.java

```diff
@@ -53,22 +44,31 @@ String createString(int kb) {
     }
 
     @TimeStep(prob = 1)
-    public void write(ThreadState state) {
+    public void set(ThreadState state) {
         atomicReference.set(atomicReference.getName());
-        state.writes++;
     }
 
-    public class ThreadState extends BaseThreadState {
-        long writes;
+    @TimeStep(prob = 0)
+    public void alter(ThreadState state) {
+        atomicReference.alter(state.identity);
+    }
+
+    @TimeStep(prob = 0)
+    public void cas(ThreadState state) {
```

Comment:
For any method that doesn't return void, it is best to return the value. This way code doesn't get optimized out.